lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/core/util/ImageBitmap.kt

```diff
@@ -0,0 +1,5 @@
+package dev.sargunv.maplibrecompose.core.util
+
+import androidx.compose.ui.graphics.ImageBitmap
+
+internal expect fun IntArray.toImageBitmap(width: Int, height: Int): ImageBitmap
```

Comment:
Hmm, why does this need separate implementation for each platform? (Also, why do I only see implementation for JS, iOS, desktop? Where's the implementation for JVM?)

There's also [ImageBitmap.toPixelMap](https://developer.android.com/reference/kotlin/androidx/compose/ui/graphics/ImageBitmap#(androidx.compose.ui.graphics.ImageBitmap).toPixelMap(kotlin.Int,kotlin.Int,kotlin.Int,kotlin.Int,kotlin.IntArray,kotlin.Int,kotlin.Int), maybe it is more convenient to work with that and doesn't require writing platform-specific code to convert it back to `ImageBitmap`?

Reply:
Desktop is JVM.

I couldn't find any other way to set the pixels of the bitmap (other than a DrawScope or something). PixelMap also appears to be read only, unless I'm missing something?

On Android we use an Android Bitmap, on the other platforms we use a Skia Bitmap. We could I suppose create a new "non-android" source set and share the Skia implementation, but it didn't seem worth it to figure that out just for one function.

Reply:
hm, I see. And on Android it is not possible to do the same as for the other implementations (so that there is only one implementation for all)?

I wonder if this would be worth a feature request at Jetpack Compose (Multiplatform).